src/components/_common/image/PublicImage.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
 
-export const PublicImage = ({ imagePath }: { imagePath: string }) => {
-    return <img src={`${process.env.APP_ORIGIN}${imagePath}`} alt={''} />;
+type Props = {
+    imagePath: string;
+    alt?: string;
+    className?: string;
+};
+
+export const PublicImage = ({ imagePath, alt = '', className }: Props) => {
```

Comment:
Burde className også defaultes slik som med alt? (Bare sånn for å være konsekvent, altså... )

Reply:
alt-attributen er et wcag krav, derfor denne har en default (tom string er greit for rene dekorative bilder, men ikke undefined). Kunne sette className = undefined, men det er ikke noe vi har gjort ellers i koden, ettersom det er implisitt.

Reply:
På en annen side, kanskje alt ikke bør ha default og ikke være optional, slik at en faktisk må ta stilling til om bildet bør ha en alt-tekst. 🤔